py/h2o_wave/cli.py

```diff
@@ -77,9 +79,21 @@ def run(app: str, no_reload: bool):
     if ext.lower() == '.py':
         app = app_path.replace(os.path.sep, '.')
 
-    uvicorn.run(f'{app}:main', host=_localhost, port=port, reload=not no_reload)
+    # Try to start Wave daemon.
+    try:
```

Comment:
Just my observation.

`try`/`finally` block feels unnecessary, but maybe it's on purpose:
1. The only function you want to `try` seems to be `Popen()`. You can move the `try` keyword right before. 🤔 
2. `finally` is executed regardless of exception. This might be what you want to run the `wave` app even if the try block fails.

Reply:
Agree, good point, thanks!